packages/gasket-plugin-analyze/lib/index.js

```diff
@@ -19,6 +19,13 @@ module.exports = {
           name: 'analyze',
           description: 'Generate analysis report of webpack bundles',
           link: 'README.md#commands'
+        }],
+        configurations: [{
+          name: 'bundleAnalyzerConfig',
+          link: 'README.md#configuration',
+          description: 'This plugin uses [webpack-bundle-analyzer] to tune both browser and server analysis reports',
+          type: 'object',
+          default: '{}'
```

Comment:
The defaults are more complex that this, i'd suggest just leave off defaults for complex objects.
```suggestion
```